<launch>
  <param name="full_arm_test/side" value="l" type="string" />
</launch>
